src/shadowbox/server/manager_metrics.ts

```diff
@@ -15,8 +15,40 @@
 import {PrometheusClient} from '../infrastructure/prometheus_scraper';
 import {DataUsageByUser, DataUsageTimeframe} from '../model/metrics';
 
+interface ServerMetricsTimeframe {
+  hours: number;
+}
+
+interface ServerMetricsServerEntry {
+  location: string;
+  asn: number;
+  asOrg: string;
+  tunnelTime?: {
```

Comment:
Define `Duration` and `Data` types, so it's clear they are the same across the interfaces.

We should probably use the same thing for the tunnel time and the time frame.